tests/conftest.py

```diff
@@ -41,7 +41,7 @@ def raise_timeout():
 
 
 @pytest.fixture
-async def test_server(loop):
+async def test_server():
```

Comment:
If the fixture is a coroutine function, it will be scheduled as a task with the `event_loop` fixture by default. We don't need to request the `event_loop` fixture explicitly.